cardano-cli/src/Cardano/CLI/Legacy/Run/Governance.hs

```diff
@@ -35,6 +36,16 @@ runLegacyGovernanceCmds = \case
     runLegacyGovernanceGenesisKeyDelegationCertificate sbe genVk genDelegVk vrfVk out
   GovernanceUpdateProposal out eNo genVKeys ppUp mCostModelFp ->
     runLegacyGovernanceUpdateProposal out eNo genVKeys ppUp mCostModelFp
+  GovernanceCreatePoll prompt choices nonce out ->
+    {- TODO replace BabbageEraOnwardsBabbage by BabbageEraOnlyBabbage when available -}
+    runGovernanceCreatePoll BabbageEraOnwardsBabbage  prompt choices nonce out
+  GovernanceAnswerPoll poll ix mOutFile ->
+    {- TODO replace BabbageEraOnwardsBabbage by BabbageEraOnlyBabbage when available -}
+    runGovernanceAnswerPoll BabbageEraOnwardsBabbage poll ix mOutFile
+  GovernanceVerifyPoll poll metadata mOutFile ->
+    {- TODO replace BabbageEraOnwardsBabbage by BabbageEraOnlyBabbage when available -}
```

Comment:
We should have to add these comments because when the `run` command's eon witness type is changed we will get a type error here and know to change it.

In any case, I originally asked for this to be changed to `BabbageEraOnly`.  I now think we should keep it as it is.  When we hard fork to Conway, users of this command should still expect this to continue working.

We can deprecate it post-Conway hard-fork.

Reply:
Removed the comments :+1:

You're right, I add in mind those `poll` commands where `babbage` only, but this is the legacy part we're speaking here; so it doesn't apply.